src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import static core.basesyntax.FigureType.CIRCLE;
+import static core.basesyntax.FigureType.ISOSCELESTRAPEZOID;
+import static core.basesyntax.FigureType.RECTANGLE;
+import static core.basesyntax.FigureType.RIGHTTRIANGLE;
+import static core.basesyntax.FigureType.SQUARE;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int RADIUS_CIRCLE = 10;
+    private static final String NAME_FIGURE = "circle";
+    private static final String WHITE_COLOR = Color.WHITE.name().toLowerCase();
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public FigureAbstract getDefaultFigure() {
+        return new Circle(NAME_FIGURE, WHITE_COLOR, RADIUS_CIRCLE);
+    }
+
+    public FigureAbstract getRandomFigure() {
```

Comment:
instead of nested if-else use switch case statement. You can use enum in switch or just integers - from 0 to number of figures